datasets/modeled/transport/dimensions/counties.csv

```diff
@@ -84,6 +84,17 @@ id,name,state
 02220,NA,NA
 02240,NA,NA
 02261,NA,NA
+02198,NA,NA
+02275,NA,NA
+02195,NA,NA
+02013,NA,NA
+02282,NA,NA
+02100,NA,NA
+02016,NA,NA
+02290,NA,NA
+02158,NA,NA
+02230,NA,NA
+02105,NA,NA
```

Comment:
Why are these all NAs? Presumably these are Alaskan counties that do have names?

Reply:
I interpreted this as Arthur saying, “the dataset has these but the project won’t use them and so we don’t care.” If we will ever use this dataset apart from the project (or as its own project), I agree that it should have names. @ahcyip Is it trivial to add these (to your branch)?